public/assests/js/message.js

```diff
@@ -200,7 +199,18 @@ function updateConversation(data) {
       var divElement1 = $("<div></div>").addClass("row message-body");
       var divElement2 = $("<div></div>").addClass("col-sm-12");
       var divElement3 = $("<div></div>");
-      var messageText = $("<div></div>").addClass("message-text").text(data[i].message);
+      var messageText = $("<div></div>").addClass("message-text");
+      
+      // Embed YouTube video if link presents in message.
+      var youtube_link = data[i].message.indexOf('https://www.youtube.com');
```

Comment:
@forsd You need to check it for youtube short url like `https://youtu.be/*` and one more thing, pls put this link in `<a>` tag so user can opn this link directly in the new tab also

Reply:
OK will do.

Reply:
Please check it.